client/src/components/Notifications/NotificationCard.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+import { BFormCheckbox } from "bootstrap-vue";
+import { defineEmits } from "vue";
```

Comment:
```suggestion
import { defineEmits } from "vue";
```
You can safely remove this import otherwise you'll get:
```
[@vue/compiler-sfc] `defineEmits` is a compiler macro and no longer needs to be imported.
```